test/integration/bigquery/folder/inspec.yml

```
@@ -0,0 +1,12 @@
+name: bigquery_folder
+depends:
+  - name: inspec-gcp
+    git: https://github.com/inspec/inspec-gcp.git
+    version: ~> 0.10.0
```

The `version` attribute is currently broken for git dependencies.

```suggestion
    tag: v0.10.0
```